### DISCUSSION TITLE
Make new sites enabled by default for sections

### DISCUSSION THREAD

**Discussion by tibbis** [craftcms/cms#15532] [Ideas]
Hi, I gradually crate new sites in the same site group and each time I have to enable the site in each section settings which is a bit time consuming. Would be great with some sort of setting to mark "new sites enabled by default" or similar! 

Thanks

**Comment by davidhellmann** [comment]
Or at least an option when Creating a site to enable it from there with a select all etc.

**Comment by alexanderbuergin** [comment]
+1
